algebird-core/src/main/scala-2.13+/com/twitter/algebird/compat.scala

```diff
@@ -0,0 +1,38 @@
+package com.twitter.algebird
+
+import scala.collection.Factory
+import scala.collection.mutable.{Builder, Map => MMap}
+
+private[algebird] class MutableBackedMap[K, V](val backingMap: MMap[K, V])
+    extends Map[K, V]
+    with java.io.Serializable {
+  override def get(key: K): Option[V] = backingMap.get(key)
+
+  override def iterator: Iterator[(K, V)] = backingMap.iterator
+
+  override def updated[B1 >: V](k: K, v: B1): Map[K, B1] = backingMap.toMap + (k -> v)
+
+  override def removed(key: K): Map[K, V] = backingMap.toMap - key
+}
+
+private[algebird] trait CompatFold {
+
+  /**
+   * Simple Fold that collects elements into a container.
+   */
+  def container[I, C[_]](implicit cbf: Factory[I, C[I]]): Fold[I, C[I]] =
+    Fold.foldMutable[Builder[I, C[I]], I, C[I]]({ case (b, i) => b += i }, { _ =>
+      cbf.newBuilder
+    }, { _.result })
+}
+
+private[algebird] trait CompatDecayedVector {
```

Comment:
Why can't this be `monoid[F, C[_]]` in 2.13? Can you comment?

Also, I see no reason to take an implicit `Ordering[Double]`. Shouldn't we just use the default one?

Reply:
hm, `monoid[F, C[_]]` will fail to compile due to ambiguous `VectorSpace[Double, C]`

https://github.com/twitter/algebird/blob/d17ba8978b9526cdf6c97e3dfae273ea6a501097/algebird-core/src/main/scala/com/twitter/algebird/VectorSpace.scala#L39-L47

With 2.13 we can make it work if we make one of them low prio.

Regarding Ordering, yup we can remove it in scala 2.13 I just carried it over;